include/uv.h

```diff
@@ -892,12 +892,16 @@ typedef struct uv_process_options_s {
   int stdio_count;
   uv_stdio_container_t* stdio;
   /*
-   * Libuv can change the child process' user/group id. This happens only when
-   * the appropriate bits are set in the flags fields. This is not supported on
-   * windows; uv_spawn() will fail and set the error to UV_ENOTSUP.
+   * Libuv can change the child process' user/group id, and supplementarty
+   * group ids. This happens only when the appropriate bits are set in the
+   * flags fields. This is not supported on windows; uv_spawn() will fail and
+   * set the error to UV_ENOTSUP.
    */
   uv_uid_t uid;
   uv_gid_t gid;
+  uv_gid_t* gids;
+  size_t gids_sz;
+
```

Comment:
Can you drop the empty line here?  Apropos the `gids_sz` field, the name is a little unidiomatic for libuv; `num_gids` or `gids_count` would be more in line with our naming conventions.